app/controllers/application_controller.rb

```diff
@@ -3,8 +3,17 @@ class ApplicationController < ActionController::Base
 
   rescue_from Pundit::NotAuthorizedError, with: :render_not_authorized
 
+  before_action :check_maintenance
   before_action :set_paper_trail_whodunnit
 
+  def check_maintenance
+    if FeatureToggle.maintenance_mode_enabled? && !%w[service-unavailable accessibility-statement privacy-notice cookies].include?(request.fullpath.split("?")[0].delete("/"))
```

Comment:
I've also allowed the user to interact with the cookie banner and see the cookies page, as well as the accessibility statement and privacy notice, as I assumed these are bits of information that we should probably make available even in maintenance mode. If you disagree and would prefer these to be hidden I can also do that.

Reply:
Note I haven't written tests for this because I wanted to get confirmation first that we did indeed want to retain these links in maintenance mode.

Reply:
I've written tests for these now.